src/renderer/state/upload/selectors.ts

```diff
@@ -460,7 +462,7 @@ export const getUploadRequests = createSelector(
           FileType.OTHER,
         originalPath: filePath,
         shouldBeInArchive: true,
-        shouldBeInLocal: true,
+        shouldBeInLocal: shouldStoreLocally ?? true,
```

Comment:
shouldStoreLocally should have a default so it is never `null` or `undefined`

Reply:
resolved in [4e933f6](https://github.com/aics-int/aics-file-upload-app/pull/116/commits/4e933f6acba2f622da8d9fe9a79869cc552ee5a0)